frontend/src/lib/k8s/node.ts

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
We have already some places where we use unique strings without uuid. See the [spawned jobs creation](https://github.com/headlamp-k8s/headlamp/blob/4d09f00a142842ef677db9fd790a7f429eb63c90/frontend/src/components/cronjob/Details.tsx#L39) for example.

Reply:
dep removed